server_extensions_test/src/main/java/com/generalbytes/batm/server/extensions/test/Tester.java

```diff
@@ -394,23 +394,27 @@ private void getRates(String name, String params) {
             IExtension extension = extensions.get(i);
             final IRateSource rs = extension.createRateSource(name + ":" + params);
             if (rs != null) {
-                final String preferredFiatCurrency = rs.getPreferredFiatCurrency();
                 final Set<String> fiatCurrencies = rs.getFiatCurrencies();
-                final Set<String> cryptoCurrencies = rs.getCryptoCurrencies();
-
-                System.out.println("Preferred Fiat Currency = " + preferredFiatCurrency);
+				StringTokenizer st = null;
```

Comment:
I guess you did this to be able to test the Syscoin rate because Tester uses the first from the list of supported coins.

However, I'd like to solve this differently (display the rate for all the supported currencies). Furthermore, your solution introduces a possible NPE on `st` variable.

Can you please rework this to displaying all the supported currencies? Thx.